File: Recursive factorial/Factorial.java
package factorial;

public class Factorial{
  // Método privado estatico
  private static Integer getFactorial(int numero){
    return (numero > 0) ? numero*getFactorial(numero - 1) : 1;
  }
  
  public static void main(String args[]){
    Integer x = 6; //Valor donde se alamacerá el número a obtenr el factorial
    System.out.println("El factorial del numero "+x.toString()+" es = "+getFactorial(x).toString()); 
  }
}

File: Recursive factorial/Factorial.py
def getFactorial(numero):
    if(numero > 0):
        return numero*getFactorial(numero - 1)
    else:
        return 1

if __name__ == '__main__':
    x = 5
    print("El factorial de "+str(x)+" es = "+str(getFactorial(x)))
